packages/api-observable/src/Base.ts

```diff
@@ -19,10 +24,43 @@ const defaultMapFn = (result: any): any =>
 // Raw base implementation for the observable API. It simply provides access to raw calls, allowing
 // decendants to make direct queries to either API methods or actual storage
 export default class ApiBase {
-  protected api: RxApiInterface;
+  protected _api: RxApiInterface;
+  protected _genesisHash: Hash;
+
+  // Promise that resolves the first time we are connected and loaded
+  whenReady: Promise<boolean>;
```

Comment:
This package is api-observable, wouldn't it make more sense to return an Observable here?

Usage:
`api.whenReady().then(...)` -> `api.whenReady().subscribe(...)`, so not a big difference for devs

Reply:
You are right.